[API] get credential endpoint documented. Credentials url conditions mentioned. (#92)
UnumID/UnumID.github.io: docs/issuance-guide.mdx

#### First Time Issuance
Some issuance calls can be made with a particular partner in mind, i.e. in a promotional flow. In these cases you have already authenticated the user and would like to issue them credentials for a particular partner.

> In this case, your partner's static `partnerUuid` can be provided to the issue [`/credentials`](/api-overview#issue-credentials) call. The resulting api response body will contain a `url` field if conditions related the partner's pre-defined <Tip type="request">credential requests</Tip> are satisfied. If present, you should redirect the user there. 

Reviewer: ```suggestion
To do this, you can provide your partner's static `partnerUuid` to the issue [`/credentials`](/api-overview#issue-credentials) call. The resulting api response body will contain a `url` field if conditions related the partner's pre-defined <Tip type="request">credential requests</Tip> are satisfied. If present, you should redirect the user there. 
```

#### Already Issued Credentials
If you have already issued credentials to a user, then [`/hasMatchingCredentials`](/api-overview#check-user-credentials) can be called with `partnerUuid`. In this case, if Verified Inc. can confirm the user has credentials to satisfy the credential request(s) in the request body, then the resulting API response body will contain a `url` which you should redirect the user to. 

> Same as in the [`/credentials`](/api-overview#issue-credentials) response, this `url` ultimately redirects the user to a page of the partner's choosing. However, the `url` first directs the user to the Verified Inc. wallet, where they explicitly consent to share credential data with the partner.

Reviewer: ```suggestion
Just as in the case of the [`/credentials`](/api-overview#issue-credentials) response, this `url` ultimately redirects the user to a page of the partner's choosing. However, the `url` first directs the user to the Verified Inc. wallet, where they explicitly consent to share credential data with the partner.
```
